### PR TITLE
Harmonize FLOSS blocks docs

### PR DESCRIPTION
**Pull request by rodrigoalmeida94**
Tracked by https://geoinformationstore.atlassian.net/browse/UP-3132

The idea of this branch is to match all the README structure in the FLOSS blocks. This is the "canary" block. Uses some parts of the README in the MODIS and in the SNAP block.

Main changes:

- [x] Remove run mechanism from makefile
- [x] Remove copy and past instructions for linux and windows
- [x] Add handle to control version number in build and push make cmd
- [x] Simplify readme with structure: **Intro, Requirements, Instructions - Installing, Testing, Pushing**

@chrieke is you can have a look and try to make use of the make utils to publish push a custom block that would be great!

Other related PRs FLOSS blocks:
- MODIS: https://github.com/up42/modis/pull/8
- Land Cover Classification: https://github.com/up42/land-cover-classification-demo/pull/7
- S2 SuperRes: https://github.com/up42/s2-superresolution/pull/14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-## General information
-
-* Block type: processing (data preparation)
-* Supported input types:
-  * Sentinel1_l1c_grd (Sentinel 1 L1C GRD in SAFE format)
-* Provider: UP42
-* Tags: SAR, radar, C-Band, imagery, preprocessing, data preparation
-
-## Description
+# SNAP polarimetric processing block
```
[the remark this answers is not in the document]
Comment (rodrigoalmeida94) [reply README.md]: Good point - i added links to the documentation page

File: README.md [hunk 2]
```diff
@@ -41,217 +40,130 @@ and delivers `up42.data.aoiclipped` as output capability.
...
-need to pass it now to `make`.
-
-```bash
-make push UID=<UID> DOCKER_TAG=<docker tag>
+make build
 ```
 
-where `<UID>` is user ID referenced above. Again using the copy &
-pasting on the clipboard.
+The e2e tests provided here make sure the blocks output conforms to the platform's
```
[the remark this answers is not in the document]
Comment (rodrigoalmeida94) [reply README.md]: Added additional explanation

File: README.md [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
-  * Sentinel1_l1c_grd (Sentinel 1 L1C GRD in SAFE format)
-* Provider: UP42
-* Tags: SAR, radar, C-Band, imagery, preprocessing, data preparation
-
-## Description
+# SNAP polarimetric processing block
+## Introduction
 
 This repository contains the code implementing a
 [block](https://docs.up42.com/getting-started/core-concepts.html#blocks)
```
[the remark this answers is not in the document]
Comment (rodrigoalmeida94) [reply README.md]: Good catch.

File: README.md [hunk 4]
```diff
@@ -1,270 +1,181 @@
...
-}
-```
-#### Get the data
-
-A radar image is needed for the block to run. Such image can be
-obtained by creating a workflow with a single **Sentinel 1 L1C GRD**
-data block and download the the result.
-
-Then create the directory `/tmp/e2e_snap_polarimetric/`:
+First login to he UP42 docker registry. `me@example.com` needs to be replaced by your **UP42 username**,
```
Comment (nekhtiari) [inline README.md]: same here: Typo

**Comment by nekhtiari** [comment]
LGTM

File: README.md [hunk 5]
```diff
@@ -1,270 +1,181 @@
...
-is `snap-polarimetric:latest`):
-
-```bash
-make build UID=<UID> DOCKER_TAG=<docker tag>
-```
+## Testing the block locally
 
-if you don't specify the docker tag, it gets the default value of `latest`.
+Before uploading the block to the UP42 platform, we encourage you to run the following local tests and validations to
+ensure that the block works as expected, conforms to the UP42 specifications and can could successfully applied in an
```
Comment (chrieke) [inline README.md]:
```suggestion
ensure that the block works as expected, conforms to the UP42 specifications and could be successfully applied in an
```

**Comment by rodrigoalmeida94** [comment]
@perusio I believe I made all the requested changes, would you agree?

**Pull request merged**